Eventonica/client/src/components/Users.jsx

```diff
@@ -0,0 +1,130 @@
+import { useState, useEffect } from "react";
+import DeleteUser from './DeleteUser';
+
+
+const Users = () => {
+  //mock users
+  const marlin = { name: 'Marlin', email: 'marlin@gmail.com', id: '1' };
+  const nemo = { name: 'Nemo', email: 'nemo@gmail.com', id: '2' };
+  const dory = { name: 'Dory', email: 'dory@gmail.com', id: '3' };
+
+  //useState for users
+  const [users, setUsers] = useState([marlin, nemo, dory])
+
+  //useState for our working components
+  const [name, setName] = useState('');
+  const [id, setID] = useState('');
+  const [email, setEmail] = useState('');
+
+  // client/src/components/Users.jsx
+  const getUsers = async () => {
+    const response = await fetch('http://localhost:4000/users');
+    const user = await response.json();
+    setUsers(user);
+  };
+
+  useEffect(() => {
+    getUsers();
+  }, []);
+
+  // Add new user
+  const handleAddUser = async (e) => {
+    e.preventDefault();
+    const newUser = { id, name, email };
+
+    const response = await fetch('http://localhost:4000/users', {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(newUser)
+    });
+    const content = await response.json();
+
+    setUsers([...users, content]);
+    setName('');
+    setEmail('');
+    setID('');
+  };
+
+  // delete new user
+  const handleDeleteUsers = async (deleteUser) => {
+    //fetches info from users for specific "deleteUser"
+    let response = await fetch(`http://localhost:4000/users/${deleteUser}`, {
+      //specifies method to remove item
+      method: "DELETE",
+    })
+    //gets json response
+    await response.json();
+    //deleting functionality 
+    const deleteUsers = users.filter((i) => i.id !== deleteUser);
+    //updates the users list
+    console.log(deleteUsers);
+    //updates list
+    setUsers(deleteUsers);
+  };
+
+  // id, name, and email are states that store what values the user types in those fields
+  // users is an array of user objects
+  // All of these states can be defined in the component
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const newUser = { id: id, name: name, email: email };
```

Comment:
You can use object shorthand here, so just `const newUser = {id, name, email};` (see object 3 here on line 14: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer)